discord/ext/commands/bot.py

```diff
@@ -220,7 +220,7 @@ def add_check(self, func, *, call_once=False):
             The function that was used as a global check.
         call_once: :class:`bool`
             If the function should only be called once per
-            :meth:`.Command.invoke` call.
+            Command.invoke call.
```

Comment:
Should this be dereferenced, or documented?

Reply:
```suggestion
            :meth:`.invoke` call.
```